SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -1990,6 +1990,7 @@
                 "clamav",
                 "cloud-hypervisor",
                 "cmake-fedora",
+                "compiler-rt",
```

Comment:
Kind general request for future reviews: please minimize overwriting commit history (through rebasing or `git commit --amend`) and instead just append new commits. This way it's a lot easier to track what changed since the last time you looked at the PR.